### PR TITLE
add bryecharts and echarts-countries-js

### PR DESCRIPTION
**Pull request by chfw**
1. bryecharts
1. echarts-countries-js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +55,8 @@ _提示：带有「🇨🇳」的项目为中文资源，或包含中文文档
 ## Extensions
 
 - 🇨🇳 [arcgis-echarts3](https://github.com/wandergis/arcgis-echarts3) - A plugin for ArcGIS JS API to load echarts3 map and Make big data visualization easier.
-- [echarts-china-cities-js](https://github.com/chfw/echarts-china-cities-js) @chfw - Packages the maps of 363 Chinese provincial cities for echarts.
+- [echarts-china-cities-js](https://github.com/chfw/echarts-china-cities-js) - Packages the maps of 363 Chinese provincial cities for echarts.
```
Comment (Ovilia) [inline README.md]: `echarts` should be `ECharts`.

File: README.md [hunk 2]
```diff
@@ -53,7 +55,8 @@ _提示：带有「🇨🇳」的项目为中文资源，或包含中文文档
...
+- [echarts-countries-js](https://github.com/chfw/echarts-countries-js) - Packages the maps of 213 countries and regions for echarts.
```
Comment (Ovilia) [inline README.md]: `echarts` should be `ECharts`.

File: README.md [hunk 3]
```diff
@@ -86,6 +89,10 @@ _提示：带有「🇨🇳」的项目为中文资源，或包含中文文档
 
 ## Languages
 
+### Brython
+
+- [bryecharts](https://github.com/chfw/bryecharts) @chfw - Draw echarts using python language in modern browsers.
```
Comment (Ovilia) [inline README.md]: This should be under `Python` section. And do not create `Brython` in table of contents either.

**Comment by chfw** [comment]
Done it

File: README.md [hunk 4]
```diff
@@ -17,12 +17,14 @@ _提示：带有「🇨🇳」的项目为中文资源，或包含中文文档
     - [React Component](#react-component)
     - [Vue Component](#vue-component)
 - [Languages](#languages)
+    - [Brython](#brython)
```
Comment (Ovilia) [inline README.md]: This should be removed.

File: README.md [hunk 5]
```diff
@@ -53,7 +55,8 @@ _提示：带有「🇨🇳」的项目为中文资源，或包含中文文档
 ## Extensions
 
 - 🇨🇳 [arcgis-echarts3](https://github.com/wandergis/arcgis-echarts3) - A plugin for ArcGIS JS API to load echarts3 map and Make big data visualization easier.
-- [echarts-china-cities-js](https://github.com/chfw/echarts-china-cities-js) @chfw - Packages the maps of 363 Chinese provincial cities for echarts.
+- [echarts-china-cities-js](https://github.com/chfw/echarts-china-cities-js) - Packages the maps of 363 Chinese provincial cities for Echarts.
```
Comment (Ovilia) [inline README.md]: `ECharts` instead of `Echarts`.

File: README.md [hunk 6]
```diff
@@ -53,7 +55,8 @@ _提示：带有「🇨🇳」的项目为中文资源，或包含中文文档
...
+- [echarts-countries-js](https://github.com/chfw/echarts-countries-js) - Packages the maps of 213 countries and regions for Echarts.
```
Comment (Ovilia) [inline README.md]: `ECharts`

**Comment by chfw** [comment]
did it again :)

**Comment by chfw** [comment]
Thanks for your review too.

**Pull request merged**